### DISCUSSION TITLE
Browse Skins in Game

### DISCUSSION THREAD

**Discussion by SlashS91** [ppy/osu#31215] [Ideas] [answered]
You know what makes a good idea? a browse category for skins in lazer (This is very useful for mobile players), I want an easy access for downloading skins instead of having to look for it on google and download some skin that keeps telling you "It may cause harm to your device".

I hope this suggestion reaches out :)

**Comment by Joehuu** [comment, marked as answer]
This was already requested in https://github.com/ppy/osu/discussions/17759, please upvote that!
